Scripts/Python/xSimpleImager.py

```diff
@@ -522,6 +523,19 @@ def OnSDLNotify(self,VARname,SDLname,playerID,tag):
             else:
                 ageVault.setDeviceInbox(ImagerName.value, ageSDL[ImagerInboxVariable.value][0], self, kSettingDeviceInbox)
 
+    def PermissionCheck(self):
+        """Age SDL Check to prevent unwanted access to imager"""
+        ageSDL = PtGetAgeSDL()
+        permissionCheck = 0
+        if ImagerPermissionCheck.value:
+            permissionCheck = ageSDL[ImagerPermissionCheck.value][0]
+        if permissionCheck >= 2: # Locked for everyone
+            return False
+        elif permissionCheck == 1 and ptVault().amOwnerOfCurrentAge(): #Hood members only
+            return True
+        elif permissionCheck <= 0: # Anyone
+            return True
```

Comment:
This feels like a good opportunity for named constants instead of magic numbers, then you can delete the inline comments, which violate PEP-8.